README.md

```diff
@@ -1,3 +1,7 @@
+Forked from [Mpv-Playlistmanager](https://github.com/jonniek/mpv-playlistmanager). Add page up/down.
+
+--------------------------------
+
```

Comment:
This attribution can be removed as both scripts are made by me and are using unlicense.

Reply:
Changes to README.md are dropped.